Dockerfile

```diff
@@ -22,6 +22,11 @@ RUN docker-php-ext-install \
 # composer
 RUN curl -sS https://getcomposer.org/installer | php && \
 	  mv composer.phar /usr/local/bin/composer
+
+#xdebug
+RUN pecl install xdebug
```

Comment:
Why do we need xdebug?

Reply:
I'm closing this and opening a different PR